mappings/net/minecraft/inventory/Inventory.mapping

```diff
@@ -22,7 +22,8 @@ CLASS net/minecraft/class_1263 net/minecraft/inventory/Inventory
 	METHOD method_5442 isInvEmpty ()Z
 	METHOD method_5443 canPlayerUseInv (Lnet/minecraft/class_1657;)Z
 		ARG 1 player
-	METHOD method_5444 getInvMaxStackAmount ()I
+	METHOD method_5444 getInvMaxCountPerStack ()I
```

Comment:
Do we need `Inv` here? (Related: #682)

Reply:
Probably not, but shouldn't the change belong to another PR ?

Reply:
I think it would be easy to remove `Inv` now that the method is being renamed anyway.

Reply:
Fair enough